HIP/hip-1064.md

```diff
@@ -0,0 +1,227 @@
+---
+hip: 1064
+title: Daily Rewards For Active Nodes
+author: Neeharika Sompalli <@Neeharika-Sompalli>
+working-group: Leemon Baird (@lbaird), Richard Bair <@rbair23>, Michael Tinker <@tinker-michaelj>
+requested-by: Hashgraph
+type: Standards Track
+category: Core
+needs-tsc-approval: Yes
+status: Draft
+last-call-date-time: 2024-04-08T07:00:00Z
+created: 2024-10-17
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/1063
+updated: 2024-10-17
+---
+
+## Abstract
+
+This HIP proposes a reward mechanism that will incentivize nodes to remain active on the network. The definition of
+active node is described below. The minimum reward per day for nodes will be a configurable property decided by the
+governance of the deployed network. This minimum reward may be adjusted based on fees already paid throughout the day.
+
+### Terminology
+
+<dl>
+<dt>Judge</dt>
+<dd>An event that wins the election to be made a judge. It must be a witness, and it will have tended to have been 
+gossiped to most of the other nodes quickly (otherwise it would have lost the election). An event reaches consensus
+when it is an ancestor of all judges in a given round. The first round where that happens is its consensus round. 
+It’s a math theorem that every round is guaranteed to have at least one judge, and a math conjecture that every round
+is guaranteed to have judges created by a supermajority of nodes (>2/3 of weight).</dd>
```

Comment:
This confuses me a bit.
Does the following express the same concepts?
```suggestion
<dd>An event wins an election to be made a judge. To win the election an event
must be a witness, and it will tend to be gossiped to most of the other nodes
quickly (otherwise it would have lost the election).

An event reaches consensus when it is an ancestor of all judges in a given round.
The first round where that happens is its consensus round. It is a mathematic
theorem that every round is guaranteed to have at least one judge, and a
mathematic conjecture that every round is guaranteed to have judges created
by a supermajority of nodes (>2/3 of consensus weight).</dd>
```

One thing that is missing:
What is the actual _criteria_ for "winning" the election?
Is it the first set of events that qualify as witnesses for the round?
Is it the first set of events that qualify as "famous witnesses"?

Reply:
Definitely confirm.  It's possible the glossary has some awkward wording, and I could easily be incorrect here.  The judges concepts are not intuitively clear, so it's best to confirm wording with our hashgraph experts before committing to it.